app/cdap/components/GridTable/types.ts

```diff
@@ -42,9 +42,22 @@ export interface IExecuteAPIResponse {
 export interface IHeaderNamesList {
   name: string;
   label: string;
-  type: Array<string | number | IRecords | boolean>;
+  type: string[];
 }
 
 export interface IObject {
   [key: string]: string | number | IRecords | boolean;
 }
+
+export interface IDataQuality {
```

Comment:
IDataQuality and IRowData looks same, can we reuse same with some generalized name?